skorch/callbacks/logging.py

```diff
@@ -205,6 +207,86 @@ def on_train_end(self, net, **kwargs):
         if self.close_after_train:
             self.experiment.stop()
 
+class WandbLogger(Callback):
+    """Logs best model and metrics to `Weights & Biases <https://docs.wandb.com/>`_
+
+    Use this callback to automatically log best trained model, all metrics from
+    your net's history, model topology and computer resources to Weights & Biases
+    after each epoch.
+
+    See `example run
+    <https://app.wandb.ai/borisd13/skorch/runs/s20or4ct/overview?workspace=user-borisd13>`_
+
+    Examples
+    --------
+    >>> # Install wandb
+    ... pip install wandb
+
+    >>> import wandb
+    >>> from skorch.callbacks import WandbLogger
+
+    >>> # Create a wandb Run
+    ... wandb_run = wandb.init()
+    >>> # Alternative: Create a wandb Run without having a W&B account
+    ... wandb_run = wandb.init(anonymous="allow)
+
+    >>> # Log hyper-parameters (optional)
+    ... wandb.config.update({"learning rate": 1e-3, "batch size": 32})
```

Comment:
Is there a way to update this using the `wandb_run` object?

Reply:
Yes! I'll update